docs/integrations/ldap/org.md

```diff
@@ -12,6 +12,14 @@ groups - directly from an LDAP compatible service. The result is a hierarchy of
 [`Group`](../../features/software-catalog/descriptor-format.md#kind-group) kind
 entities that mirror your org setup.
 
+## Supported vendors
+
+Currently, Backstage only supports the following LDAP vendors:
```

Comment:
I'm not sure it's only those. I think it's more like we support pretty much any LDAP vendor that doesn't deviate from some common patterns, right? As in, the FreeIPA system obviously had a super vendor specific ID field offered.

Could we word it more like this?

`Backstage in general supports OpenLDAP compatible vendors, as well as Active Directory and FreeIPA. If you are using a vendor that does not seem to be supported, please [file an issue](https://github.com/backstage/backstage/issues/new?assignees=&labels=enhancement&template=feature_template.md).`

Reply:
I was already doubtful about the "default" LDAP implementation. This is a great rewording. Updated the PR